extensions/amp-analytics/0.1/vendors/kenshoo.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const KENSHOO_CONFIG = /** @type {!JsonObject} */ ({
+  'vars': {
+    'clientId': 'CLIENT_ID(ken_amp_id)',
+    'channelClickId': 'QUERY_PARAM(gclid)',
+    'tid': 'KT-XXXXX-XXX',
+    'userId': '<USER_ID>',
+  },
+  'requests': {
+    'host': 'https://amp.xg4ken.com',
+    'parameters':
+      'ampcid=${clientId}' +
+      '&chcid=${channelClickId}' +
+      '&tid=${tid}' +
+      '&uid=${userId}' +
+      '&domain=${canonicalHostname}',
+    'landingPage': '${host}/amp/v1/match?${parameters}',
+  },
+  'triggers': {
+    'trackLandingPage': {
+      'enabled': 'QUERY_PARAM(gclid)',
+      'on': 'visible',
+      'request': 'landingPage',
+    },
+  },
+  'transport': {
+    'beacon': false,
+    'xhrpost': false,
+    'image': true,
```

Comment:
Does your server support xhr or beacon? We prefer to use these methods, and will fallback if browser does not have support?

Reply:
@calebcordry To be honest - I followed the examples I saw in the code and on the AMP online [documentation](https://www.ampproject.org/docs/analytics/deep_dive_analytics) :smile:  ,
However, our server does support both `beacon` and `xhrpost`. (and of course also `image`)

I see that the [default](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/vendors.js#L108) value is **true** for all, so basically I can remove this section. right?
(we don't need to override the `useBody` for `beacon` and for the `xhrpost` because all the parameters are on the query string, and we don't use the `suppressWarnings` for the `image`)

Reply:
Yes if you are fine with the default recommended behavior, you can remove this section. Thanks